src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.spec.ts

```diff
@@ -59,6 +61,7 @@ describe('DotEditPageWorkflowsActionsComponent', () => {
     describe('button enabled', () => {
         beforeEach(() => {
             spyOn(dotWorkflowService, 'getContentWorkflowActions').and.callThrough();
+            component.ngOnChanges();
```

Comment:
we should not call ngOnChanges directly in the test, here you can see how do that:

https://github.com/dotCMS/core-web/blob/master/src/app/portlets/content-types/layout/content-types-layout.component.spec.ts#L33

Reply:
done